scripts/minify-synthetix-contract.js

```diff
@@ -0,0 +1,22 @@
+const fs = require('fs');
+const ethers = require('ethers');
+
+module.exports = function generate({ networks, out }) {
+	networks.forEach((network) => {
+		const deployment = require(`synthetix/publish/deployed/${network}/deployment.json`);
+
+		Object.keys(deployment.targets).forEach((key) => {
+			const { name, source, address } = deployment.targets[key];
+			deployment.targets[key] = { name, source, address };
+		});
+
+		Object.keys(deployment.sources).forEach((key) => {
+			const source = deployment.sources[key];
+			const iface = new ethers.utils.Interface(source.abi);
+			const abi = iface.format(ethers.utils.FormatTypes.full);
+			deployment.sources[key] = { abi };
```

Comment:
Do we do this to strip out stuff that we don't need? I would expect the a oi to already be there